### PR TITLE
feat: Add `vue-tour` support for FLINT UI

### PR DESCRIPTION
**Pull request by iamrajiv**
## Description

Vue Tour is a lightweight, simple, and customizable guided tour plugin for use with Vue.js. It provides a quick and easy way to guide your users through your application.

https://user-images.githubusercontent.com/42106787/140501342-8262b736-37e1-4bd9-8d78-440939cddf45.mov

### CODE REVIEW

**Comment by aornugent** [comment]
Hi @shubhamkarande13 - what do you mean? There is nothing in the repository that indicates a blocking feature, so I don't understand why Rajiv's work should not be merged? 🤔

File: flint.ui/src/views/Landing.vue [hunk 1]
```diff
@@ -64,101 +34,149 @@
...
+          buttonPrevious: 'Previous',
+          buttonNext: 'Next',
+          buttonStop: 'Finish'
+        }
+      },
+      steps: [
+        {
+          target: '[data-v-step="1"]',
+          content:
+            'Point is a sample run of FLINT. To learn how to run the simulation using the FLINT UI dashboard follow the tour.',
```
Comment (aornugent) [inline flint.ui/src/views/Landing.vue:156]: I would move the second this sentence to a new prompt (later), and then change the text to 

> Point is a simple three-pool example of a stock and flow modell running in FLINT

File: flint.ui/src/views/Landing.vue [hunk 2]
```diff
@@ -64,101 +34,149 @@
...
+          params: {
+            placement: 'top'
+          }
+        },
+        {
+          target: '[data-v-step="3"]',
+          content: 'Click on this button to run The Generic Carbon Budget Model (GCBM) simulation.',
+          params: {
+            placement: 'top'
+          }
```
Comment (aornugent) [inline flint.ui/src/views/Landing.vue:173]: Then add one more prompt here that goes back to the Point button and says:

> Follow the tour by clicking here to learn how to run a simulation using the FLINT UI.

**Comment by iamrajiv** [comment]
@aornugent Implemented thee requested changes.

**Comment by iamrajiv** [comment]
@aornugent Implemented the requested changes.

**Review by aornugent — APPROVED** [review]

**Comment by aornugent** [comment]
These look good to me! Nice work @iamrajiv 

If we don't hear back from @shubhamkarande13 I think it's ok to merge.

**Pull request merged**